src/orderbook.js

```diff
@@ -164,7 +201,15 @@ OrderBook.EVENTS = [
   'offer_changed', 'offer_funds_changed'
 ];
 
-OrderBook.DEFAULT_TRANSFER_RATE = 1000000000;
+OrderBook.DEFAULT_TRANSFER_RATE = Amount.from_json(1000000000);
+
+OrderBook.NOTIFY_THROTTLE_TIME = 4000;
```

Comment:
I've changed formula in last commit, so now it looks like this
AUTOBRIDGE_CALCULATE_THROTTLE_TIME @ 4000:

Compute Time | Multiplier | Throttle Timeout
----------------------|---------------|------------------------
1999|1|4000
2000|2|8000
4000|3|12000

if we change AUTOBRIDGE_CALCULATE_THROTTLE_TIME @ 1000:
it will be

Compute Time | Multiplier | Throttle Timeout
----------------------|---------------|------------------------
499|1|1000
500|2|2000
1000|3|3000
1500|4|4000

I think that will be ok